score-lang-cli/src/main/java/org/openscore/lang/cli/SlangCLI.java

```diff
@@ -85,12 +86,12 @@ public String run(
         if (!triggerAsync) {
             StopWatch stopWatch = new StopWatch();
             stopWatch.start();
-            id = scoreServices.triggerSync(compilationArtifact, mergedInputs, systemProperties);
+            id = scoreServices.triggerSync(compilationArtifact, mergedInputs, systemProperties, quiet);
             stopWatch.stop();
-            return triggerSyncMsg(id, stopWatch.toString());
+            return quiet ? StringUtils.EMPTY : triggerSyncMsg(id, stopWatch.toString());
         }
-        id = scoreServices.trigger(compilationArtifact, mergedInputs, systemProperties);
-        return triggerAsyncMsg(id, compilationArtifact.getExecutionPlan().getName());
+        id = scoreServices.trigger(compilationArtifact, inputs, systemProperties);
```

Comment:
I think here it was mergedInputs instead of inputs. Fixed this in cli_extentions_support